docs/log-ingestion/fluentd.md

```diff
@@ -1,31 +1,27 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Fluentd
 
 _Ingesting logs into Siglens using Fluentd_
 
-Fluentd is an open-source data collector that unifies data collection and consumption for better use and understanding of data. It's designed to handle various types of data sources with different types and formats, making it a popular choice for logging solutions.
-
-In this guide, we will walk through the process of using Fluentd to send logs to Siglens.
-
-## 1. Install Fluentd
-
-- Follow the instructions to install the Fluentd package from the [official docs](https://docs.fluentd.org/installation).
+### 1. Install Fluentd
 
-  - [Debian/Ubuntu](https://docs.fluentd.org/installation/install-by-deb#installing-fluent-package)
-  - [macOS](https://docs.fluentd.org/installation/obsolete-installation/treasure-agent-v4-installation/install-by-dmg-td-agent-v4)
-  - [Windows](https://docs.fluentd.org/installation/install-by-msi)
+Install <a href="https://docs.fluentd.org/installation" target="_blank">Fluentd</a> on your server
 
-## 2. Configure Fluentd
+### 2. Configure Fluentd
 
-- Create a Fluentd config file with the below [sample configuration](#sample-configuration-file).
-- If you are looking for a sample log dataset you can download it from [here](https://github.com/siglens/pub-datasets/releases/download/v1.0.0/2kevents.json.tar.gz) and untar it.
-
-### Sample Configuration File
+Download the sample events file using the following command:
+```bash
+curl -s -L https://github.com/siglens/pub-datasets/releases/download/v1.0.0/2kevents.json.tar.gz -o 2kevents.json.tar.gz && tar -xvf 2kevents.json.tar.gz
+```
 
-```conf
+Create a fluentd.conf file:
+```xml title="fluentd.conf"
 <source>
   @type tail
-  path D:\Siglens\data\2kevents.json
-  pos_file D:\Siglens\fluentd_logs\2kevents.json.pos
+  path /Users/username/logstash/2kevents.json # Path to the log file
+  pos_file /Users/username/logstash/2kevents.json  # Path to the position file
```

Comment:
Is this `pos_file` correct (it's the same as the `path` file)? I'm not too familiar with fluentd, so maybe this is correct, I just wanted to double check